client/src/components/NavBar/User/index.tsx

```diff
@@ -136,43 +140,47 @@ const NavBarUser = ({ shareFiles, isSkeleton }: Props) => {
           )}
         </div>
         {shareFiles?.length ? <ShareButton files={shareFiles} visible /> : ''}
-        <span>
-          {isSkeleton ? (
-            <div className="flex items-center gap-1.5 text-gray-500">
-              <div className="bg-gray-700 rounded-full h-10 w-10" />
-              <ChevronDownFilled />
-            </div>
-          ) : (
-            <DropdownWithIcon
-              items={[
-                {
-                  text: 'Settings',
-                  icon: <Cog />,
-                  type: MenuListItemType.DEFAULT,
-                  onClick: () => setSettingsOpen(true),
-                },
-                // {
-                //   text: 'My Collections',
-                //   icon: <Collections />,
-                //   type: MenuListItemType.DEFAULT,
-                // },
-                {
-                  text: 'Report a bug',
-                  icon: <Bug />,
-                  type: MenuListItemType.DEFAULT,
-                  onClick: () => setBugReportModalOpen(true),
-                },
-                // {
-                //   text: 'Sign out',
-                //   icon: <DoorRight />,
-                //   type: MenuListItemType.DEFAULT,
-                // },
-              ]}
-              icon={<Person />}
-              dropdownBtnClassName="-mr-4"
-            />
-          )}
-        </span>
+        {isRepoManagementAllowed ? (
+          <span>
+            {isSkeleton ? (
+              <div className="flex items-center gap-1.5 text-gray-500">
+                <div className="bg-gray-700 rounded-full h-10 w-10" />
+                <ChevronDownFilled />
+              </div>
+            ) : (
+              <DropdownWithIcon
+                items={[
+                  {
+                    text: 'Settings',
+                    icon: <Cog />,
+                    type: MenuListItemType.DEFAULT,
+                    onClick: () => setSettingsOpen(true),
+                  },
+                  // {
+                  //   text: 'My Collections',
+                  //   icon: <Collections />,
+                  //   type: MenuListItemType.DEFAULT,
+                  // },
+                  {
+                    text: 'Report a bug',
```

Comment:
We'll want report bug functionality enabled even when repo management is disabled